apps/src/applab/ApplabVisualizationColumn.jsx

```diff
@@ -47,7 +47,13 @@ var ApplabVisualizationColumn = React.createClass({
             screenIds={this.props.screenIds}
             onScreenCreate={this.props.onScreenCreate} />
         }
-        <PhoneFrame showFrame={showFrame} isDark={this.props.isRunning}>
+        <PhoneFrame
+            showFrame={showFrame}
+            isDark={this.props.isRunning}
+            showSelector={this.props.interfaceMode === applabConstants.ApplabInterfaceMode.DESIGN}
+            screenIds={this.props.screenIds}
+            onScreenCreate={this.props.onScreenCreate}
```

Comment:
Ideally screenIs would live in redux and onScreenCreate would be an action. However, this becomes a little more challenging, since changing screenIds also results in changes to non-react managed DOM. For now I decided to leave this as is as a result.